app/agents/voice/automatic/processors/llm_spy.py

```
@@ -34,6 +35,23 @@
 from app.core.logger import logger
 
 from ..features.text_sanitizer.tts_sanitizer import sanitize_markdown
+from app.agents.voice.automatic.audio.audio_manager import get_audio_manager
+
+def _stop_audio_immediately(context: str = "unknown") -> bool:
+    """INSTANT audio stopping using simplified AudioManager API."""
+    audio_manager = get_audio_manager()
+    if audio_manager and (audio_manager.user_has_input or audio_manager.is_playing or audio_manager.is_in_delay_period):
+        # Stop function call set audio if active
+        if audio_manager.function_call_set_audio_playing:
+            asyncio.create_task(audio_manager.stop_function_call_set_audio())
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)